tests/unit/test_gff3_parser.py

```diff
@@ -0,0 +1,102 @@
+"""
+Data-driven tests for rna quantification.
+"""
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import os
+import ga4gh.gff3 as gff3
+import unittest
+
+
+class TestGff3ParserOnTypicalFile(unittest.TestCase):
+    """
+    Data driven unit tests for the GFF3 parser
+    """
+    def setUp(self):
```

Comment:
I don't think you want to bring in another GFF3 parser as a
dependency.

The reason a GGF3 parser was written is the two GFF3 parser
available, one was inadequate toy with no tests and not well
written. The other had a very heavy weight dependency
(biopython).  If there has been a usable GFF3 parser, we would
have used it.

One approach I use for test code that works on files is to
create expected files and diff them with test results. In this
cases, the write function of the GFF3 can be use to create a
file.  While it will not be directly comparable to the source
due to lack of ordering constraints, it can then be manually
validated in various ways.  The manual validation can then be
done (inspection, run through web validation, etc).  The
expected file is then check in.


Jerome Kelleher <notifications@github.com> writes:
> In tests/unit/test_gff3_parser.py:
> Usually with data driven tests we want to compare the results of two difference
> parsing approaches. VCF is the canonical example: we use PyVCF to parse the
> VCFs and we make sure our code agrees with it. Can we import some other GFF
> parser to parse the file for us?
> 
> —
> Reply to this email directly or view it on GitHub.*
>